src/dualscreen.tsx

```diff
@@ -402,3 +433,30 @@ export class DualScreen extends KalturaPlayer.core.BasePlugin implements IEngine
     this.eventManager.destroy();
   }
 }
+
+// TEMP - for QA drop only
+// @ts-ignore
+window.addSlide = (url, startTime = Math.round(kalturaPlayer.currentTime), endTime = kalturaPlayer.currentTime + 10) => {
+  let slideTrack;
+  // @ts-ignore
+  for (let track of kalturaPlayer.getVideoElement().textTracks) {
+    if (track.kind === 'metadata' && track.label === 'KalturaCuePoints' && track.language === 'slides') {
+      slideTrack = track;
+    }
+  }
+  if (!slideTrack) {
+    // @ts-ignore
+    slideTrack = kalturaPlayer.getVideoElement().addTextTrack('metadata', 'KalturaCuePoints', 'slides');
+  }
+  const cue = new VTTCue(startTime, endTime, '');
```

Comment:
not supported in ie11 see https://github.com/kaltura/playkit-js/blob/master/src/track/text-track.js#L102